docs/upgrade/2.10.1_to_2.11.0.rst

```diff
@@ -0,0 +1,93 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.10.1 to 2.11.0
+=============================
+
+Preparing for the Ubuntu 24.04 (Noble) migration
+------------------------------------------------
+
+The 2.11.0 release includes a number of features that will help ensure
+your SecureDrop server is prepared for the automated migration to Ubuntu 24.04
+(Noble) in early 2025.
+
+After updating to SecureDrop 2.11.0 using the steps below, it's important
```

Comment:
Maybe also mention that admins/journalists may see a banner indicating that there are problems blocking migration?